### PR TITLE
#2206: Create Organisation form wording clarified

### PR DESCRIPTION
**Pull request by i-hardy**
#2206

### CODE REVIEW

File: AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Organization/Edit.cshtml [hunk 1]
```diff
@@ -57,17 +57,17 @@
         <h3>Organization Information</h3>
 
         <div class="form-group">
-            <label asp-for="LogoUrl" class="col-md-2 control-label"></label>
+            <label asp-for="Name" class="col-md-2 control-label">Organisation Name</label>
```
Comment (stevejgordon) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Organization/Edit.cshtml]: On this the label text will actually get rendered on the backend based on the properties of our model class. PR #2252 is a good example of this. Are you able to adjust in this case?

File: AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Shared/EditorTemplates/Location.cshtml [hunk 2]
```diff
@@ -2,14 +2,14 @@
 <div>
     <input type="hidden" asp-for="Id" />
     <div class="form-group">
-        <label asp-for="Address1" class="col-md-2 control-label"></label>
+        <label asp-for="Address1" class="col-md-2 control-label">Address Line 1</label>
```
Comment (stevejgordon) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Shared/EditorTemplates/Location.cshtml]: I think these can be affected on AllReadyApp/Web-App/AllReady/Areas/Admin/ViewModels/Shared/LocationDisplayViewModel.cs

File: AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Organization/Edit.cshtml (hunk 1 above)
Comment (i-hardy) [reply AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Organization/Edit.cshtml]: No problem

**Comment by MisterJames** [comment]
@stevejgordon and @i-hardy would you folks be able to follow up on this one so we can get it merged? Thanks!

**Pull request closed without merging**